### ISSUE TITLE
Priority 1 accessibility issue: Back to top link focus

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6546]
**What**

There are back-to-top links included on the page which do not take screen reader focus to the designated location.

When activating the ‘back to top’ link with screen reader software and navigating in reading mode via cursor keys, focus is not taken to the top of the page. This can be navigationally disorientating as users would expect to have their focus taken to the top of the page upon activating the link, regardless of the navigation method employed.

**Screen reader comments:**
“When I activated the ‘Back to top’ link, Screen Reader focus remained on the ‘Back to top’ link. This is confusing because the description of the link suggests that users will be taken to the top of the page. Ensuring that Screen Reader focus follows the link to the target destination will avoid confusion and disorientation.”

**Solution:**
Ensure that focus is taken to the appropriate location upon activating the link both for keyboard focus and programmatic focus, allowing screen reader users to navigate via the cursor keys.

Code Ref(s): #__next > div:nth-child(2) > div.footer.row.bg-black.relative > div > div.footer__bottom > nav > ul > li:nth-child(6) > a
<a href="#top" class="footer__hygiene-link footer__hygiene-link--back-to-top"> <span>Back to top</span>
<div class="icon icon--270">
<svg class="icon__svg" aria-hidden="true">[...]</svg><
/div> </a>

Code Ref(s): #top
<html id="top" lang="en" class="enhanced fonts-loaded">[...]</html>

WCAG Reference:
2.4.3 Focus Order (Level A)
Understanding Focus Order | How to Meet Focus Order

**Where**

URL: https://wellcomecollection.org
Page: Wellcome Collection | A free museum and library exploring health and human experience
